target/linux/ath79/image/nand.mk

```diff
@@ -1,4 +1,17 @@
 # attention: only zlib compression is allowed for the boot fs
+define Build/kt-headers
```

Comment:
I could not find any other device using this header format, so I think `kt-headers` is too inclusive. How about including the vendor or the device name in the function name?

And you should add this function above the zlib comment line.

Reply:
I named it because the images of devices from other manufacturers (e.g., Mercury) also use this type of header.
However, since there is little chance that other devices will be supported, it seems good to change its name.

Reply:
> I named it because the images of devices from other manufacturers (e.g., Mercury) also use this type of header.

Could you name a few? (for cross-check)

Reply:
> Could you name a few? (for cross-check)

"GiGA WiFi home" is manufactured by two manufacturers, "Dongwon T&I" and "Mercury".
There are five models: DW01-410H, DW02-412H, KM05-410H, KM06-506H, and KM06-704H.
They all use the same firmware made by KT.

Reply:
Are you sure they share the same FW? KM06-506H is based on a Realtek CPU and its firmware image header does not seem to deviate from what is commonly used on other Realtek devices.

Reply:
> Are you sure they share the same FW? KM06-506H is based on a Realtek CPU and its firmware image header does not seem to deviate from what is commonly used on other Realtek devices.

I found a kernel module used for realtek chips in DW02-412H's firmware, so I thought all of these devices used the same firmware. I think I might have made a hasty conclusion...